Decred Wallet/Features/Send/ConfirmToSendFundViewController.swift

```diff
@@ -91,20 +91,20 @@ class ConfirmToSendFundViewController: UIViewController, UITextFieldDelegate {
         if let exchangeRate = self.exchangeRate {
             let sendingAmountUsd = self.unsignedTxSummary.dcrAmount.multiplying(by: exchangeRate)
             let sendingAmountUsdText = Utils.styleAttributedString(
-                " (\(sendingAmountUsd.round(4).formattedWithSeparator))",
+                " ($\(sendingAmountUsd.round(4).formattedWithSeparator) USD)",
                 font: UIFont(name: "SourceSansPro-Regular", size: 25),
                 color: subTextColor
             )
             amountText.append(sendingAmountUsdText)
             
             let feeUsd = self.unsignedTxSummary.dcrFee.multiplying(by: exchangeRate)
             feeText.append(
-                Utils.styleAttributedString(" (\(feeUsd.round(8).formattedWithSeparator))", color: subTextColor)
+                Utils.styleAttributedString("DCR ($\(feeUsd.round(4).formattedWithSeparator) USD)", color: subTextColor)
```

Comment:
` DCR` should be appended to `feeText` regardless of whether or not exchange rate data is available. Same for sending amount.

Also, should not combine `$` and `USD`:
```suggestion
                Utils.styleAttributedString(" ($\(feeUsd.round(4).formattedWithSeparator))", color: subTextColor)
```